extensions/data-workspace/src/common/dataWorkspaceExtension.ts

```diff
@@ -0,0 +1,12 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { IExtension } from 'dataworkspace';
+
+export class DataWorkspaceExtension implements IExtension {
+	constructor(public getProjectsInWorkspace: () => vscode.Uri[]) {
```

Comment:
why not take workspaceService as a parameter as we discussed earlier?

Reply:
oops forgot about that. Fixed now